src/languages/es-ES.ts

```diff
@@ -2475,6 +2475,28 @@ export default class extends Language {
 		COMMAND_COINFLIP_WIN_DESCRIPTION: (result, wager) => `The coin was flipped, and it showed ${result}. ${wager ? `You guessed correctly and won ${wager} ${SHINY}` : 'You got it right'}!`,
 		COMMAND_COINFLIP_LOSE_DESCRIPTION: (result, wager) => `The coin was flipped, and it showed ${result}. You didn\'t guess corectly ${wager ? `and lost ${wager} ${SHINY}.` : ''}.`,
 		COMMAND_COINFLIP_NOGUESS_DESCRIPTION: result => `The coin was flipped, and it showed ${result}.`,
+		COMMAND_HIGHERLOWER_DESCRIPTION: 'Play a game of Higher/Lower',
+		COMMAND_HIGHERLOWER_EXTENDED: builder.display('higherlower', {
+			extendedHelp: `Higher Lower is a game of luck. I pick 2 numbers, and you have to guess if the 2nd is **higher** or **lower** then the 1st, using the ⬆ or ⬇ emojis.
+			Your winnings increase as you progress through the rounds, and can cashout any time pressing the 💰 reaction.
+			Be warned tho! The further you go, the more chances you have to lose the winnings.`
+		}),
+		COMMAND_HIGHERLOWER_LOADING: `${LOADING} Starting a new game of Higher Lower`,
```

Comment:
```suggestion
		COMMAND_HIGHERLOWER_LOADING: `${LOADING} Comenzar un nuevo juego de Mayor Menor`,
```

Reply:
```suggestion
		COMMAND_HIGHERLOWER_LOADING: `${LOADING} Comenzar un nuevo juego de Mayor/Menor`,
```

Also add the `/` in the English one.